content/courses/offline-transactions/durable-nonces.md

```diff
@@ -115,30 +115,30 @@ From the
 
 ### Considerations
 
-Durable transactions should be treated with care, and are why you should always
+Durable transactions should be treated with care, which is why you should always
 trust the transactions you sign.
 
-Say you blindly signed a malicious durable transaction. This transaction signs
-away 500 SOL to the attacker, and changes the nonce authority to said attacker.
-Let's say you don't have this much yet, but in the future, you would. This is
-insidious, as the attacker would wait to cash this check as soon as your balance
-goes above 500 SOL. And you'll have no recollection of what you clicked on. It
-can lay dormant for days, weeks, or years.
+For example, imagine you blindly signed a malicious durable transaction. This
+transaction signs away 500 SOL to the attacker and changes the nonce authority
+to the attacker as well. Let's say you don't have this amount yet, but in the
+future, you will. This is insidious, as the attacker would wait to cash this
+check as soon as your balance exceeeds 500 SOL. And you'll have no recollection
+of what you clicked on. It can remain dormant for days, weeks, or years.
 
-This is not meant to provoke hysteria, just as a PSA of what's possible. This is
-why you should only put into hot wallets what you're willing to lose and don't
-sign with your cold wallet.
+This is not meant to provoke hysteria, but serves as a PSA of what's possible.
```

Comment:
```suggestion
This is not meant to provoke hysteria, but serves to show what's possible.
```

PSAs are US-specific.

Reply:
This lesson goes from 'you are the developer' to 'you are the user'. We should fix that.

Maybe as developers, we tell people their durable nonce trandsactions may be flagged by wallets, and then explain that durable nonces are often used for malicious transactions?

Reply:
I have revised the section to maintain a consistent perspective, addressing both developers and users. The updated text now informs developers to educate users about the potential risks and how to mitigate them.

Please let me know if there are any further adjustments needed.